examples/dataproc-gcs-connector/README.md

```diff
@@ -0,0 +1,89 @@
+# Testing GCS Connector for Dataproc
+
+The Google Cloud Storage connector for Hadoop (HCFS) enables running [Apache Hadoop](http://hadoop.apache.org/) or [Apache Spark](http://spark.apache.org/) jobs directly on data in [GCS](https://cloud.google.com/storage) by implementing the Hadoop FileSystem interface. The connector is developed as an [open source project](https://github.com/GoogleCloudDataproc/hadoop-connectors) and comes preinstalled on Cloud Dataproc clusters.
```

Comment:
Should we move this guidde to https://github.com/GoogleCloudDataproc/hadoop-connectors project?  There it will be much easier to access and be able to be updated by experts who develop this tool. WDYT?
CC: @jaketf

Reply:
We planned on putting it in PSO since it incorporates different Dataproc repos (GoogleCloudDataproc/hadoop-connectors, GoogleCloudDataproc/initalization-actions) and is a general solution. It would be a good idea to at least include a reference to this solution from the Dataproc GitHub though.
CC @jaketf